api/types.go

```diff
@@ -94,11 +94,14 @@ type ChatRequest struct {
 	Format string `json:"format"`
 
 	// KeepAlive controls how long the model will stay loaded into memory
-	// followin the request.
+	// following the request.
 	KeepAlive *Duration `json:"keep_alive,omitempty"`
 
 	// Options lists model-specific options.
 	Options map[string]interface{} `json:"options"`
+
+	// OpenAI indicates redirection from the compatibility endpoint.
```

Comment:
Adding this flag isn't quite right. Instead, let's fix this in all of the `/api/generate`, `/api/chat` and `/v1/chat/completions` endpoints in one PR as it is broken in all three.

Caching is an implementation detail and shouldn't affect these fields in our API (it does today):

* `prompt_eval_count`
* `eval_count`